src/control/lib/hardware/hwloc/bindings.go

```diff
@@ -554,7 +554,8 @@ func (o *object) linkSpeed() (float64, error) {
 	if pciDevAttr == nil {
 		return 0, errors.Errorf("device %q attrs are nil", o.name())
 	}
-	return float64(pciDevAttr.linkspeed), nil
+	// Attribute in Giga units, return raw transactions per sec.
+	return float32(pciDevAttr.linkspeed) * 1e+9, nil
```

Comment:
nit: The multiplier should be defined as a const.

Reply:
done